pyanaconda/ui/gui/spokes/network.py

```diff
@@ -1206,6 +1134,300 @@ def _try_disconnect(obj, callback):
             log.debug("%s", e)
 
 
+class ConfigureWirelessNetworksDialog(GUIObject):
+    builderObjects = ["configure_wireless_network_dialog",
+                      "liststore_configure_wireless_network"]
+    mainWidgetName = "configure_wireless_network_dialog"
+    uiFile = "spokes/network.glade"
+
+    def __init__(self, data, nm_client):
+        GUIObject.__init__(self, data)
+        self._nm_client = nm_client
+
+    def initialize(self):
```

Comment:
Do we need the initialize method? This can be done in `__init__`.

Reply:
Done.